ui/admin-portal/src/app/components/candidate-opp/view-candidate-opp-from-url/view-candidate-opp-from-url.component.spec.ts

```diff
@@ -14,6 +14,7 @@ import {
   OpportunityStageNextStepComponent
 } from "../../util/opportunity-stage-next-step/opportunity-stage-next-step.component";
 import {RouterLinkStubDirective} from "../../login/login.component.spec";
+import {CUSTOM_ELEMENTS_SCHEMA} from "@angular/core";
```

Comment:
It would to document the need for this somewhere - maybe just with a link to external doc - eg https://stackoverflow.com/questions/39428132/custom-elements-schema-added-to-ngmodule-schemas-still-showing-error